config/thermofloor/heatit021-v1.92.xml

```diff
@@ -0,0 +1,221 @@
+Air min. limit 5°C (5°C…40°C)
+Air max. limit 35°C (5°C…40°C)
+Regulator min. limit 0 (0…9)
+Power regulator P02 (P00…P10) cycle of 30 minutes
+Temperature setback Adjustable within temperature range.
+Temperature setback activation Pilot wire/Z-Wave user interface
+Hysteresis: 0,5°C
+Sensor values: 0°C 29,1 kOhm
+10°C 18,6 kOhm
+15°C 15,1 kOhm
+20°C 12,2 kOhm
+25°C 10,0 kOhm
+Switch: Single-pole
+Standard Color White RAL 9003 – Black RAL 9011
+Approvals CE – FIMKO - Z-Wave Plus
+		</MetaDataItem>
+		<MetaDataItem name="ExclusionDescription">INCLUSION/EXCLUSION
+To include the thermostat to your home automation gateway, press Center (confirm) for 10 seconds.
+The display will show OFF. Press Right (down) 4 times till you see Con in the display.
+Now start add device in your home automation software. Start inclusion mode by pressing Center (confirm) for approximately
+2 seconds. The inclusion/exclusion icon will appear in the display. Confirmation will show Inc/EcL in the display. If inclusion
+fails, Err (error) will appear.
+Leave programming mode by choosing ESC in menu. Your thermostat is ready for use with default settings.</MetaDataItem>
+		<MetaDataItem name="ProductManual">https://products.z-wavealliance.org/ProductManual/File?folder=&amp;filename=MarketCertificationFiles/1182/Manual_Multireg%20Z-Wave_Ver2015-A.pdf</MetaDataItem>
+		<MetaDataItem id="0001" name="FrequencyName" type="0001">CEPT (Europe)</MetaDataItem>
+		<MetaDataItem name="Name">Heatit Z-Wave</MetaDataItem>
+		<MetaDataItem id="0001" name="Identifier" type="0001">TF021</MetaDataItem>
+		<MetaDataItem name="ResetDescription">FACTORY RESET - RES
+By pressing buttons Left and Center (up and confirm) for 20 seconds, the thermostat will perform a complete factory reset.
+NB! Please use this procedure only when the primary controller is missing or otherwise inoperable.</MetaDataItem>
+		<MetaDataItem name="InclusionDescription">INCLUSION/EXCLUSION
+To include the thermostat to your home automation gateway, press Center (confirm) for 10 seconds.
+The display will show OFF. Press Right (down) 4 times till you see Con in the display.
+Now start add device in your home automation software. Start inclusion mode by pressing Center (confirm) for approximately
+2 seconds. The inclusion/exclusion icon will appear in the display. Confirmation will show Inc/EcL in the display. If inclusion
+fails, Err (error) will appear.
+Leave programming mode by choosing ESC in menu. Your thermostat is ready for use with default settings.</MetaDataItem>
+		<MetaDataItem name="WakeupDescription">If Er2 appears in the display, touch any button to restart the display.</MetaDataItem>
+		<ChangeLog>
+			<Entry author="Pål Kristensen - paalkr77@gmail.com" date="11 January 2021" revision="1">Initial release</Entry>
+		</ChangeLog>
+	</MetaData>
+
+	<CommandClass id="32">
+		<Compatibility>
+			<Mapping>37</Mapping>
+		</Compatibility>
+	</CommandClass>
+	<CommandClass id="49">
```

Comment:
I cannot get OZW to assign the correct labels for the sensors. Command Class 49 always ends up like this in the OZW cache
```
			<CommandClass id="49" name="COMMAND_CLASS_SENSOR_MULTILEVEL">
				<Compatibility />
				<State />
				<Instance index="1" endpoint="2" />
				<Instance index="2" endpoint="3" />
				<Instance index="3" endpoint="4" />
				<Value type="decimal" genre="user" instance="1" index="1" label="External Floor Temperature Sensor" units="C" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="-11.6">
					<Help>Internal Air Temperature Sensor Value</Help>
				</Value>
				<Value type="decimal" genre="user" instance="2" index="1" label="External Floor Temperature Sensor" units="C" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="0.0">
					<Help>Internal Air Temperature Sensor Value</Help>
				</Value>
				<Value type="decimal" genre="user" instance="3" index="1" label="External Floor Temperature Sensor" units="C" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="0" max="0" value="24.6">
					<Help>Internal Air Temperature Sensor Value</Help>
				</Value>
			</CommandClass>
```
			
Tot very important though, mostly annoying.